.prettierignore

```diff
@@ -3,3 +3,4 @@ package.json
 package-lock.json
 packages/cli/snippets/next.js
 packages/cli/src/generators/templates/
+.yarn
```

Comment:
`prettier` was hanging for a long time on my first pre-commit because it was trying to prettier all of `yarn` 🙈